packages/block-editor/src/hooks/color.js

```diff
@@ -243,11 +243,17 @@ export function ColorEdit( props ) {
 
 	return (
 		<ColorPanel
-			enableContrastChecking={ ! gradient && ! style?.color?.gradient }
+			enableContrastChecking={
+				// Turn on contrast checker for web only since it's not supported on mobile yet.
+				Platform.OS === 'web' && ! gradient && ! style?.color?.gradient
+			}
 			clientId={ props.clientId }
 			settings={ [
 				{
-					label: __( 'Text Color' ),
+					label:
+						Platform.OS === 'web'
```

Comment:
Yup, this is what design indicated, but maybe we can unify it @iamthomasbishop ?

Reply:
What do you think @iamthomasbishop ? could we have the same texts as the web editor? It'd show up like this:

<img src="https://user-images.githubusercontent.com/4885740/80602931-0a0e6400-8a30-11ea-9eba-06b8605d9c64.png" width="250" />

Reply:
That makes sense. I think my initial mocks removed the word "color" because I felt it was redundant (_color_ settings, followed by text _color_, then background _color_), but for the sake of consistency that's fine. Does each sub-sheet then inherit those labels (when you go to the text color sub-sheet, is the title "Text color" or "Text"?)? (Maybe that's a question for @lukewalczak )

Reply:
> Does each sub-sheet then inherit those labels (when you go to the text color sub-sheet, is the title "Text color" or "Text"?)? (Maybe that's a question for @lukewalczak )

Yes, it does. Headers will be then `Background Color` and `Text Color`

Reply:
Perfect! 🙌